extensions/blocks/podcast-player/style.scss

```diff
@@ -100,30 +101,41 @@ $player-float-background: $light-gray-200;
 		height: $cover-image-size;
 	}
 
-	.jetpack-podcast-player__title {
-		display: flex;
-		flex-direction: column;
-		margin: 0;
-		overflow: hidden;
-	}
-
-	.jetpack-podcast-player__current-track-title {
-		font-size: $track-title-font-size;
-		margin: 0 0 $track-title-b-margin;
-	}
-
-	.jetpack-podcast-player__podcast-title {
-		font-size: $podcast-title-font-size;
-		color: $text-color;
-		margin: 0;
-	}
-
-	a.jetpack-podcast-player__podcast-title {
-		text-decoration: none;
-
-		&:hover,
-		&:focus {
-			color: $text-color-hover;
+	// Increasing specificity on the header titles and links for more consistency across themes and editor canvas
+	.jetpack-podcast-player__header {
+		
+		.jetpack-podcast-player__title {
+			display: flex;
+			flex-direction: column;
+			margin: 0;
+			overflow: hidden;
+			letter-spacing: 0; // Fixes 2020 compressed text
```

Comment:
Let's use Twenty Twenty's spelled-out name to indicate we're referring to the theme and not the year.

Reply:
Done 👍 

-